pkgs/misc/emulators/wine/base.nix

```diff
@@ -7,10 +7,15 @@
 
 with import ./util.nix { inherit lib; };
 
+let configureFlags' = configureFlags ++ [ "--disable-tests" ];
+in
+
 stdenv.mkDerivation ((lib.optionalAttrs (! isNull buildScript) {
   builder = buildScript;
 }) // rec {
```

Comment:
It would be cleaner to remove `rec` here and set `configureFlags = [ "--disable-tests" ] ++ configureFlags`.